package.json

```diff
@@ -63,49 +66,51 @@
     "react-focus-lock": "^2.5.2",
     "react-helmet-async": "^1.0.9",
     "react-input-autosize": "^3.0.0",
-    "react-number-format": "^4.6.4",
-    "react-responsive-carousel": "^3.2.19",
+    "react-number-format": "^4.7.3",
+    "react-responsive-carousel": "^3.2.20",
     "react-router-dom": "^5.2.0",
     "react-toastify": "^7.0.4",
-    "web3modal": "^1.9.3"
+    "web3modal": "^1.9.4"
   },
   "devDependencies": {
     "@aragon/os": "^4.4.0",
     "@nomiclabs/hardhat-ethers": "^2.0.0",
     "@openzeppelin/contracts": "^4.2.0",
-    "@testing-library/cypress": "^7.0.6",
+    "@testing-library/cypress": "^8.0.0",
     "@testing-library/jest-dom": "^5.14.1",
     "@testing-library/react": "^12.0.0",
-    "@testing-library/user-event": "^13.1.9",
+    "@testing-library/user-event": "^13.2.1",
     "@typechain/ethers-v5": "^7.0.1",
-    "@types/chai": "^4.2.19",
-    "@types/faker": "^5.5.6",
-    "@types/jest": "^26.0.15",
-    "@types/lodash": "^4.14.170",
+    "@types/chai": "^4.2.21",
+    "@types/faker": "^5.5.7",
+    "@types/jest": "^26.0.24",
+    "@types/lodash": "^4.14.172",
     "@types/mocha": "^8.2.2",
-    "@types/node": "^16.0.0",
-    "@types/react": "^17.0.13",
-    "@types/react-dom": "^17.0.8",
-    "@types/react-input-autosize": "^2.2.0",
-    "@types/react-router-dom": "^5.1.7",
+    "@types/node": "^16.4.13",
+    "@types/react": "^17.0.16",
+    "@types/react-dom": "^17.0.9",
+    "@types/react-input-autosize": "^2.2.1",
+    "@types/react-router-dom": "^5.1.8",
     "api3-dao": "git+https://github.com/api3dao/api3-dao.git#0b8bb17fbda0161d5d9fff00ed4562b91cc27af6",
     "chai": "^4.3.4",
-    "chalk": "^4.1.1",
-    "cypress": "^7.6.0",
+    "chalk": "^4.1.2",
+    "concurrently": "^6.2.1",
+    "cypress": "^8.2.0",
     "cypress-plugin-tab": "^1.0.5",
     "dotenv": "^10.0.0",
     "faker": "^5.5.3",
-    "hardhat": "^2.4.1",
-    "husky": "^7.0.0",
-    "lint-staged": "^11.0.0",
+    "hardhat": "^2.6.0",
+    "husky": "^7.0.1",
+    "lint-staged": "^11.1.2",
     "node-sass": "^6.0.1",
     "npx": "^10.2.2",
     "prettier": "^2.3.2",
     "react-scripts": "4.0.3",
     "replace-in-file": "^6.2.0",
     "sort-package-json": "^1.50.0",
-    "ts-node": "^10.0.0",
-    "typechain": "^5.1.1",
+    "start-server-and-test": "^1.13.1",
```

Comment:
Where is this used?

Reply:
Allows you to call `yarn start-test` which is what I use to run cypress tests on CI (but I am not very successful in it as of now :D )